lambdas/get-users-id-request/src/index.ts

```diff
@@ -0,0 +1,65 @@
+import {Qualification, User} from "../../../common/help-on-spot-models/dist";
+import Request from "../../../common/help-on-spot-models/dist/entity/Request";
+import {Database} from "../../../common/help-on-spot-models/dist/utils/Database";
+import {LambdaResponse, lambdaResponse} from "../../../common/help-on-spot-models/dist/utils/lambdaResponse";
+
+export interface LambdaInputEvent {
+    pathParameters: {
+        userId: string,
+        radius?: string,
+        requestType?: string,
+        location?: { lat: number, long: number }
+    }
+}
+
+/* TODO: this should probably be done as part of the DB query. But since this will change anyway once we change to
+* actual locations I guess it is good enough atm.
+*/
+function matchesUserQualifications(request: Request, userQualifications: Qualification[] | undefined): boolean {
+    if (!request.qualifications || request.qualifications.length === 0) {
+        return true
+    }
+    const userKeys = userQualifications!.map(r => r.key)
```

Comment:
I would name this variable `userQualificationKeys`. I also prefer longer var names in array methods (e.g.: `qualification => qualification.key`)